src/Stratis.Bitcoin/Utilities/DBreezeSerializer.cs

```diff
@@ -32,7 +32,7 @@ internal byte[] Serializer(object obj)
         {
             IBitcoinSerializable serializable = obj as IBitcoinSerializable;
             if (serializable != null)
-                return serializable.ToBytes(options: this.NetworkOptions);
+                return serializable.ToBytes(consensusFactory: this.Network.Consensus.ConsensusFactory);
```

Comment:
Here you could just pass the `this.Network` instead of `this.Network.Consensus.ConsensusFactory`. It would make the code easier to read ("Ok, here he's serializing base on the network") and you'd also have  the whole network objec to play with inside the method.